src/components/MetadataPanel/style.css

```
@@ -63,7 +63,7 @@
 
 .row {
     display: flex;
-    font-weight: 100;
+    font-weight: 400; 
```

nittiest nit: stray space
```suggestion
    font-weight: 400;
```

---

ty!